minifi-nar-bundles/minifi-framework-bundle/minifi-framework/minifi-resources/src/main/resources/conf/bootstrap.conf

```diff
@@ -28,6 +28,16 @@ conf.dir=./conf
 # How long to wait after telling MiNiFi to shutdown before explicitly killing the Process
 graceful.shutdown.seconds=20
 
+# The location for the configuration file
+nifi.minifi.config=./conf/config.yml
+
+# Notifiers to use for the associated agent, comma separated list of class names
+#nifi.minifi.notifier.components=org.apache.nifi.minifi.bootstrap.configuration.FileChangeNotifier
+
+# File change notifier configuration for path of the file to monitor and the associated polling period
+#nifi.minifi.notifier.file.config.path=
```

Comment:
Was a bit unsure about how the mechanics of this would work from a general interface standpoint.  What I was thinking was that new configurations could be pushed in file form by dropping them in a given watch directory and then the rest would behave as currently specified.  If we are watching the file that is the config itself, this could be a bit messy.

I think watching the file itself could also be fine, but would think that there would need to be some indicator within the handleChange to a) not ingest the file as an InputStream and b) not actually write the file out.

Good either way, just want to ensure the various implementations do not have to cater to that only of the file.